src/fractalHelpers.erl

```diff
@@ -111,7 +109,7 @@ computeRowOfFractalData(FractalAlg, {PixelY,ImgY}, XList, RowOfFractalData, Conf
     %% otherwise pop off one x value, compute data, insert answer in RowOfFractalData, and recurse
     [ {PixelX, RealX} | NewXList ] = XList,   % pop off first x value, remainder is used for next iteration
     %% compute fractal value for x,y
-    IterCount = computeIterationValue(FractalAlg,
+    IterCount = compute_points:compute_iteration_value(FractalAlg,
                                       maps:get(cReal,ConfigMap), %since julian, C remains constant
                                       maps:get(cImaginary,ConfigMap), %since julian, C remains constant
```

Comment:
According to **Elvis**:
> Line 114 is too long:                                       maps:get(cImaginary,ConfigMap), %since julian, C remains constant.